content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -0,0 +1,94 @@
+
+{{< img src="profiler/guide-monolithic-outliers/4-outliers-monolith-flame-graph.png" alt="Your image description" style="width:100%;" >}}
+
+To better understand why the `GET /store_history` endpoint in particular is using more CPU, look at the table on the right-hand side, where you can see the endpoint is second from the top. Select just that row to focus the flame graph specifically on the CPU utilization caused by the `GET /store_history` endpoint. 
+
+Because you are investigating resource usage per request, also change the value in the drop-down menu at top of the table to `CPU Time per Endpoint Call`, to show the average resource usage per call to that endpoint instead of the average resource usage per minute.
+
+## Comparing flame graphs
+
+Now that the graph shows data for the right time and endpoint, you probably have enough data to understand what is causing the spike in CPU utilization. But if not, you can compare the flame graph for the spike with a time when utilization was more acceptable.
+
+To see if there are differences in which methods are using a lot of CPU time between a spike and normal usage, click **Compare** (on the far right side of the search field) and select `Previous 15 minutes`. This opens the Comparison view. 
+
+The view shows two graphs, labeled `A` and `B`, each representing a time range for CPU utilization per call for `GET /store_history`. Adjust the time selector for `A` so that it is scoped to a period with low CPU utilization per call:
+
+{{< img src="profiler/guide-monolithic-outliers/5-outliers-monolith-compare-flame-graphs.png" alt="Your image description" style="width:100%;" >}}
+
+The comparison shows the different methods causing CPU utilization on the right side (timeframe `B` during the spike) that are not used at all in timeframe `A` (normal CPU). The culprit, `Product.loadAssets(int)`, is causing the spikes. 
+
+To fix the problem, optimize the method. Looking at the method code, the signature is `Product(int id, String name, boolean shouldLoadAssets)` and you do not need to load assets for the response to the `GET /store_history` endpoint. This implies that there is a bug further up the call stack that improperly instructs the `Product` constructor to load assets.
+
+Fix that bug and verify that the spikes go away, using the timeseries graphs discussed earlier.
+
+## Isolate the impact of operations (Java)
+
+There are other attributes available in the profiler. For example, you can filter and group a flame graph by operation names, rather than by functions or threads. For a monolithic application this can be more helpful, as it can more directly state which resources are utilizing more CPU, even if they are shared between endpoints.
```

Comment:
```suggestion
There are other attributes available in the profiler. For example, you can filter and group a flame graph by operation names, rather than by functions or threads. For monolithic applications, this can more clearly identify CPU-intensive resources, even if they are shared between endpoints.
```